dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/monitor/AdvancedRumMonitor.kt

```diff
@@ -13,11 +13,14 @@ import com.datadog.android.rum.internal.domain.event.ResourceTiming
 import com.datadog.android.rum.model.ViewEvent
 import com.datadog.tools.annotation.NoOpImplementation
 
+@SuppressWarnings("ComplexInterface")
 @NoOpImplementation
 internal interface AdvancedRumMonitor : RumMonitor {
 
     fun resetSession()
 
+    fun sendEmptyEvent()
```

Comment:
Instead of empty event, maybe we can add some meaning here, e.g.:`fun notifyWebViewRumEventIngested` ? (ok this one is probably a bit too verbose, maybe `fun addWebViewEvent()` ?